### DISCUSSION TITLE
Disable ALT-Arrows shortcuts

### DISCUSSION THREAD

**Discussion by SpidFightFR** [labwc/labwc#1008] [Q&A] [answered]
Hey lads, i hope you're doing great.

I got an issue with the Alt-arrows bindings, basically, i'm used to use those shortcuts for basically every apps (alt-left for going in the previous page, alt-right to go back to the next page, alt-up to go to the above directory in a File manager), and labwc overwrite them for the snap to edge feature... 

How can i unbind those shortcuts please ? I tried to re-bind them to an empty command, it stops the thing with the window moving, but i still can't use the old shortcuts i mentioned before.

Thanks by advance for the help. Amazing windows manager btw !

**Comment by stefonarch** [comment, marked as answer]
The first answer here: https://labwc.github.io/faq.html

**Reply by SpidFightFR** [reply]
Oh wow it was right in front of me the whole time LOL !
Thank you for your help !

I added this in `rc.xml`:

```
<keybind key="A-Left"/>
<keybind key="A-Right"/>
<keybind key="A-Up"/>
<keybind key="A-Down"/>
```

This indeed solves the issue.
